css-size-adjust-1/Overview.bs

```diff
@@ -163,21 +210,52 @@ Size adjustment control: the 'text-size-adjust' property {#adjustment-control}
 
   <dl dfn-for="text-size-adjust" dfn-type="value">
     <dt><dfn>auto</dfn></dt>
-      <dd>Renderers must use the <a href="#default-adjustment">default size adjustment</a> when displaying on a small device.</dd>
+      <dd>Renderers must use the <a href="#default-adjustment">default size adjustment</a> that is best tailored to the individual user's and device's viewing conditions.</dd>
+      <p class="note">Note: On most desktop machines, the best adjustment is no adjustment.</p></dd>
     <dt><dfn>none</dfn></dt>
-      <dd>Renderers must not do size adjustment when displaying on a small device.</dd>
+      <dd>Renderers must not adjust sizes.</dd>
     <dt><dfn>&lt;percentage&gt;</dfn></dt>
-      <dd>When displaying on a small device, renderers must not do size adjustment but instead the computed value of 'font-size' must be multiplied by this percentage.
-      <p class="note">Note: This means that 'text-size-adjust: 100%;' is equivalent to 'text-size-adjust: none;'.</p></dd>
+      <dd>Provided as a hint to the User Agent about a suggested adjustment amount. User agents may consider this percentage value when computing an adjusted text size.</dd>
```

Comment:
I don't think this is a sufficient definition.  It needs to say what the percentage means (which the old text did -- although it's possible it was wrong), even if it then says that UAs don't have to honor it.  It would be better to say what the inputs into the decision of whether to honor it are.